lib/Runtime/SerializableFunctionFields.h

```diff
@@ -71,6 +71,7 @@ PRIVATE_FIELDS
     DECLARE_SERIALIZABLE_FIELD(bool, m_hasFinally, Bool);
     DECLARE_SERIALIZABLE_FIELD(bool, hasScopeObject, Bool);
     DECLARE_SERIALIZABLE_FIELD(bool, hasCachedScopePropIds, Bool);
+    DECLARE_SERIALIZABLE_ACCESSOR_FIELD(uint, ForInLoopDepth, UInt32);
```

Comment:
ForInLoopDepth is the real depth.  ProfiledForInLoopCount is restrict to uint16 depth.   We will stop generating profiled byte code for depth > uint16.